Sphinx 6.0.0: Theme Error: Reason: UndefinedError("'logo' is undefined")

After upgrade to `sphinx==6.0.0`, a `ThemeError` is raised. This error disapear when using another theme, so I think it's related to `pydata_sphinx_theme`.

```
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [  0%] api/astropop.astrometry.AstrometricSolution                                                                                                 
Theme error:
An error happened in rendering the page api/astropop.astrometry.AstrometricSolution.
Reason: UndefinedError("'logo' is undefined")
make: *** [Makefile:45: html] Error 2
```

--- a/src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/navbar-logo.html
+++ b/src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/navbar-logo.html
@@ -7,9 +7,9 @@
 {% endif %}
 <a class="navbar-brand logo" href="{{ href }}">
   {# get all the brand information from html_theme_option #}
-  {% set is_logo = logo or theme_logo.get("image_light") or theme_logo.get("image_dark") %}
-  {% set image_light = theme_logo.get("image_light") or logo %}
-  {% set image_dark = theme_logo.get("image_dark") or logo %}
+  {% set is_logo = logo_url or theme_logo.get("image_light") or theme_logo.get("image_dark") %}
+  {% set image_light = theme_logo.get("image_light") or logo_url %}
+  {% set image_dark = theme_logo.get("image_dark") or logo_url %}
   {% set image_light = image_light if image_light.startswith("http") else pathto('_static/' + image_light, 1) %}
   {% set image_dark = image_dark if image_dark.startswith("http") else pathto('_static/' + image_dark, 1) %}
   {% set alt = theme_logo.get("alt_text", "Logo image") %}

